app/main.py

```diff
@@ -2,5 +2,16 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cached = {}
+
+    def inner(*args, **kwargs) -> None:
+        if args in cached.keys():
```

Comment:
no need to use keys() method here, it will search through keys by default
in similar case too

Reply:
Got it! Thanks.